core/lib/makeswift/components/site-theme/components/index.ts

```diff
@@ -18,7 +18,7 @@ export default {
   footer,
   header,
   logo,
-  priceLabel,
```

Comment:
The `<SiteTheme>` component auto-generates CSS variable names based on the corresponding prop names. The [CSS variable prefix for the `<PriceLabel>` component is `price`](https://github.com/bigcommerce/catalyst/blob/canary/core/vibes/soul/primitives/price-label/index.tsx#L30), not `price-label`:

<img width="1014" alt="CSS vars" src="https://github.com/user-attachments/assets/7be441fd-b39a-49c5-a467-8fbc3397cfc0" />